src/Symfony/Component/Workflow/Tests/RegistryTest.php

```diff
@@ -101,14 +102,14 @@ public function testAllWithMultipleMatchWithSuccess()
         $this->assertSame('workflow3', $workflows[1]->getName());
     }
 
-    public function testAllWithNoMatch()
+    public function testAllWithNoMatch(): void
     {
         $workflows = $this->registry->all(new \stdClass());
         $this->assertIsArray($workflows);
         $this->assertCount(0, $workflows);
     }
 
-    private function createWorkflowSupportStrategy($supportedClassName)
+    private function createWorkflowSupportStrategy($supportedClassName): MockObject|WorkflowSupportStrategyInterface
```

Comment:
Not sure about it. this function always returns a MockObject, if you want to be more precise you can use the phpdoc

Reply:
this should be an intersection type, not a union type